default/python/universe_generation/galaxy.py

```diff
@@ -109,15 +422,13 @@ def spiral_galaxy_calc_positions(positions, arms, size, width):
             break
 
         if not attempts:
-            print "Spiral galaxy shape: giving up on placing star", i,\
-                  ", can't find position sufficiently far from other systems"
+            print "Spiral galaxy shape: giving up on placing star {}, can't find position sufficiently far from other systems.".format(i)
```

Comment:
The print statement output was wider than 79 characters.  It seemed wrong to break them, when they would not be broken in the log output.

Fixed in any case.